src/core/view/background/PlainBackgroundView.cpp

```diff
@@ -7,11 +7,16 @@
 
 using namespace xoj::view;
 
-PlainBackgroundView::PlainBackgroundView(double pageWidth, double pageHeight, Color backgroundColor):
-        BackgroundView(pageWidth, pageHeight), backgroundColor(backgroundColor) {}
+PlainBackgroundView::PlainBackgroundView(double pageWidth, double pageHeight, Color backgroundColor,
+                                         std::optional<Recolor> recolor):
+        BackgroundView(pageWidth, pageHeight), backgroundColor(backgroundColor), recolor(recolor) {}
 
 void PlainBackgroundView::draw(cairo_t* cr) const {
-    Util::cairo_set_source_rgbi(cr, this->backgroundColor);
+    if (recolor) {
+        Util::cairo_set_source_rgbi(cr, this->recolor->dark);  // TODO(recolor)
+    } else {
+        Util::cairo_set_source_rgbi(cr, this->backgroundColor);
+    }
```

Comment:
All this is only needed since in the stacked view, the layers (which are not the background) get the `backgroundColor` as default background.

Should we simply use the dark color here (like I'm doing currently, since it's the easy way out and one could argue by doing the recolor, the user usually wants to use the dark color as background color anyhow), or do a proper recoloring of the background color being set? (would involve manually inverting, "scaling" and "moving" the background color)